src/main/java/uk/gov/hmcts/divorce/noticeofchange/event/SystemRequestNoticeOfChange.java

```diff
@@ -113,6 +119,11 @@ public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
 
         String sysUserToken = idamService.retrieveSystemUpdateUserDetails().getAuthToken();
         String s2sToken = authTokenGenerator.generate();
+        boolean isApplicant1 = APPLICANT_1_SOLICITOR.getRole()
+                .equals(details.getData().getChangeOrganisationRequestField().getCaseRoleId().getRole());
+
+        notificationDispatcher.sendNOC(nocCitizenToSolsNotifications, details.getData(),
+                beforeDetails.getData(), details.getId(), isApplicant1, ORG_REMOVED);
```

Comment:
I think calling notificationDispatcher here in the request event will mean we send the org removed notification to the old solicitor twice? 🤔 It looks like the code you've added in [SystemApplyNoticeOfChange](https://github.com/hmcts/nfdiv-case-api/pull/3848/files#diff-71225c811691792a2c8a98b10b93e13fac31d29e0e68261b46516c362ff8cdcfR97-R98) that passes `NEW_DIGITAL_SOLICITOR_NEW_ORG` is enough to send all the notifications?

Reply:
Call to `NotificationDispatcher.sendNOC` has if condition that decides who to send notifications to. I think that was done mainly due to Caseworker NoC. We call the if condition based on `noticeType`

Reply:
Do you mean here?:
<img width="730" alt="image" src="https://github.com/hmcts/nfdiv-case-api/assets/159780146/99434421-bf93-4de6-a2a7-04a771d8aab1">
We call sendNOC in applySystemNoticeOfChange with `NEW_DIGITAL_SOLICITOR_NEW_ORG` and call it again in the submitted callback of SystemRequestNoticeOfChange with `ORG_REMOVED` - won't that mean the second if statement evaluates to true both times and we repeat the notification to the old solicitor?